docs/content/misc/tasks.md

```diff
@@ -139,7 +139,8 @@ The Hadoop Index Task is used to index larger data sets that require the paralle
 |type|The task type, this should always be "index_hadoop".|yes|
 |spec|A Hadoop Index Spec. See [Batch Ingestion](../ingestion/batch-ingestion.html)|yes|
 |hadoopCoordinates|The Maven \<groupId\>:\<artifactId\>:\<version\> of Hadoop to use. The default is "org.apache.hadoop:hadoop-client:2.3.0".|no|
-
+|hadoopDependencyCoordinates|A JSON array of Hadoop dependency coordinates that will override the default Hadoop coordinates and `hadoopCoordinates`.|no|
```

Comment:
hadoopDependencyCoordinates is a list and is the preferred one, hadoopCoordinates is a string and is for backwards compatibility. IMO we should remove hadoopCoordinates from the docs.